frontend/components/filter/FilterTags.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import { Button } from "@/components/Button";
+import { TFilterId, TFilterItem } from "@/types/filter";
+
+type FilterTagButtonProp = TFilterItem & {
+    onPress: () => void;
+};
+
+export const FilterTagButton = ({ id: title, onPress, selected = false }: FilterTagButtonProp) => {
+    return (
+        <Button
+            title={title}
+            onPress={onPress}
+            containerStyle={[
+                styles.filterButton,
+                selected ? styles.filterHighlightedButton : styles.filterUnhighlightedButton,
+            ]}
+            textStyle={[styles.filterText]}
+        />
+    );
+};
+
+const styles = StyleSheet.create({
+    filterButton: {
+        paddingVertical: 3,
+        paddingHorizontal: 8,
+        borderRadius: 12,
+        borderWidth: 1,
+        borderColor: "#000000",
+    },
+    filterHighlightedButton: {
+        backgroundColor: "#FFCF0F",
```

Comment:
We should try to break all of our colors out into a Colors.tsx file or something similar